packages/airnode-protocol/contracts/rrp/requesters/RrpBeaconServerV0.sol

```diff
@@ -1,10 +1,9 @@
 // SPDX-License-Identifier: MIT
```

Comment:
Replaced Whitelist + WhitelistRolesWithManager with WhitelistWithManager, similar to DapiServer